dtoolkit/transformer/_decorator.py

```diff
@@ -1,18 +0,0 @@
-from functools import wraps
-
-from ._util import transform_array_to_frame
-
-
-def frame_in_frame_out(func):
-    """
-    Wrap :func:`~dtoolkit.transformer._util.transform_array_to_frame` to
-    decorator.
-    """
-
-    @wraps(func)
-    def decorator(self, X):
-        X_new = func(self, X)
```

Comment:
Why does there remove the decorator?

This line code couldn't handle all considerations.